### PR TITLE
Update performance with Java/Android examples

### PR DESCRIPTION
**Pull request by ElieDeBrauwer**
performance.mdown lacks examples for several languages (only js, objc and switf are present today). This patch add all missing Android examples making the Android documents more easy to read.

### CODE REVIEW

File: en/common/performance.mdown [hunk 1]
```diff
@@ -169,6 +193,9 @@ query.containedIn("state", ["SignedUp", "Verified"]);
 ```common-swift
 query.whereKey("state", containedIn: ["SignedUp", "Verified"])
 ```
+```common-java
+query.whereNotContainedIn("state", Arrays.asList("SignedUp", "Verified"));
```
Comment (grantland) [inline en/common/performance.mdown]: I think you mean `whereContainedIn` :smile:

File: en/common/performance.mdown [hunk 2]
```diff
@@ -221,6 +261,10 @@ PFQuery *query = [PFUser query];
 var query = PFUser.query()
 query.whereKey("state", containedIn: ["Invited", "Blocked"])
```
Comment (grantland) [inline en/common/performance.mdown]: mind doing a ninja edit while you're here and fixing this to be `notContainedIn`?

File: en/common/performance.mdown [hunk 3]
```diff
@@ -456,6 +553,19 @@ query.countObjectsInBackgroundWithBlock {
   }
 }
 ```
+```common-java
+ParseQuery<ParseObject> query = ParseQuery.getQuery("Review");
+// movieId corresponds to a given movie's id
+query.whereEqualTo("movie", movieId;
```
Comment (grantland) [inline en/common/performance.mdown]: Looks like you're missing the end `)` here

**Comment by grantland** [comment]
I found a few typos that need to be fixed, but this looks to be a great addition! Thanks for putting in the effort to keep the docs in parity across all platforms :smile:

**Comment by amo-ej1** [comment]
Thanks for the review, I updated my commit to incorporate your suggestions.

**Comment by grantland** [comment]
Thanks for the awesome contribution!

**Pull request merged**